event/src/main/java/com/facebook/airlift/event/client/EventDataType.java

```diff
@@ -136,12 +135,12 @@ public void writeFieldValue(JsonGenerator jsonGenerator, Object value)
         }
     },
 
-    DATETIME(DateTime.class) {
```

Comment:
lets not remove joda DateTime.
Just add additional support for ZoneDateTime

Reply:
Added ZONEDDATETIME